src/distraction_manager.cpp

```diff
@@ -0,0 +1,216 @@
+        uistate.distraction_noise,
+        uistate.distraction_pain,
+        uistate.distraction_attack,
+        uistate.distraction_hostile_close,
+        uistate.distraction_hostile_spotted,
+        uistate.distraction_conversation,
+        uistate.distraction_asthma,
+        uistate.distraction_dangerous_field,
+        uistate.distraction_weather_change,
+        uistate.distraction_hunger,
+        uistate.distraction_thirst,
+    };
+
+    input_context ctx{ "DISTRACTION_MANAGER" };
+    ctx.register_action( "QUIT" );
+    ctx.register_action( "HELP_KEYBINDINGS" );
+    ctx.register_cardinal();
+    ctx.register_action( "CONFIRM" );
+
+    ui.on_redraw( [&]( const ui_adaptor & ) {
+        // Draw border
+        draw_border( w_border, BORDER_COLOR, _( "Distractions manager" ) );
+        mvwputch( w_border, point( 0, iHeaderHeight - 1 ), c_light_gray, LINE_XXXO );
+        mvwputch( w_border, point( 79, iHeaderHeight - 1 ), c_light_gray, LINE_XOXX );
+        mvwputch( w_border, point( 61, FULL_SCREEN_HEIGHT - 1 ), c_light_gray, LINE_XXOX );
+        wnoutrefresh( w_border );
+
+        // Draw header
+        werase( w_header );
+        fold_and_print( w_header, point_zero, getmaxx( w_header ), c_white,
+                        configurable_distractions[currentLine].second.c_str() );
+
+        // Draw horizontal line and corner pieces of the table
+        for( int x = 0; x < 78; x++ ) {
+            if( x == 60 ) {
+                mvwputch( w_header, point( x, iHeaderHeight - 2 ), c_light_gray, LINE_OXXX );
+                mvwputch( w_header, point( x, iHeaderHeight - 1 ), c_light_gray, LINE_XOXO );
+            } else {
+                mvwputch( w_header, point( x, iHeaderHeight - 2 ), c_light_gray, LINE_OXOX );
+            }
+        }
+
+        wnoutrefresh( w_header );
+
+        // Clear table
+        for( int y = 0; y < iContentHeight; y++ ) {
+            for( int x = 0; x < 79; x++ ) {
+                if( x == 60 ) {
+                    mvwputch( w, point( x, y ), c_light_gray, LINE_XOXO );
+                } else {
+                    mvwputch( w, point( x, y ), c_black, ' ' );
+                }
+            }
+        }
+
+        draw_scrollbar( w_border, currentLine, iContentHeight, number_of_distractions, point( 0,
+                        iHeaderHeight + 1 ) );
+
+        calcStartPos( startPosition, currentLine, iContentHeight, number_of_distractions );
+
+        for( int i = startPosition; i < number_of_distractions; ++i ) {
+            const nc_color line_color = i == currentLine ? hilite( c_white ) : c_white;
+            const nc_color status_color = distractions_status[i] ? c_light_green : c_red;
+            const std::string status_string = distractions_status[i] ? _( "Enabled" ) : _( "Disabled" );
+
+            // Print distraction types
+            mvwprintz( w, point( 1, i - startPosition ), line_color, "%s",
+                       configurable_distractions[i].first.c_str() );
```

Comment:
```suggestion
                       _( configurable_distractions[i].first.c_str() ) );
```
I think this is required for translation to work. The string is marked for extraction, but it is not actually translated anywhere.